server/manager/src/main/java/org/apache/accumulo/manager/FateServiceHandler.java

```diff
@@ -771,8 +771,6 @@ public void executeFateOperation(TInfo tinfo, TCredentials c, long opid, FateOpe
               "Length of requested split exceeds tables configured max, see warning in logs for more information.");
         }
 
-        manager.requestUnassignment(extent, opid);
```

Comment:
I didnt' replace this with a call to `requestUnloadTablet` as the FATE operations in the `isReady` method should make this call if the tablet has a location and retry.